processor/sourcemap/_meta/fields.yml

```diff
@@ -6,14 +6,20 @@
       dynamic: false
       type: group
       fields:
-        - name: app_name
-          type: keyword
-          description: >
-            The name of the app this sourcemap belongs to.
-        - name: app_version
-          type: keyword
+        - name: app
+          type: group
           description: >
-            App version.
+            App fields.
+          fields:
+            - name: name
+              type: keyword
+              description: >
+                The name of the app this sourcemap belongs to.
+            - name: version
+              type: keyword
+              description: >
+                App version.
+
```

Comment:
Tiny thing: Could we put the `sourcemap` field in here so it shows up in the docs but it will still not get indexed?

Reply:
I don't know. How do you do that? I can't find anywhere else where we do that.

Reply:
No that's not possible, see https://github.com/elastic/apm-server/issues/28

Reply:
right 👍